### ISSUE TITLE
Does anyone got on awards from api data? 

### ISSUE THREAD

**Issue by toppyc4** [adrianhajdin/project_travel_advisor#44]
Seem like data that provide to me have an awards key but it contain empty arrays. 
Does anyone facing this issue or know how to fix it?

**Comment by joshuanarvaez** [comment]
I completed this project a few months ago and had no issue displaying the awards. However, I revisited it today and can't get the awards to display. I also get practically no data from Hotels. I think it's because the API is deprecated but I might be wrong.

**Comment by SHIVENDRA8004** [comment]
@joshuanarvaez Yes, you are correct i am also facing some issues while fetching data of hotels and awards, so i added conditions in the code if there is any awards then only it will display unless it will act like ""
